nestkernel/eprop_archiving_node.h

```diff
@@ -175,8 +341,12 @@ class EpropArchivingNodeReadout : public EpropArchivingNode< HistEntryEpropReado
   //! Copy constructor.
   EpropArchivingNodeReadout( const EpropArchivingNodeReadout& );
 
-  //! Create an entry in the eprop history for the given time step and error signal.
-  void write_error_signal_to_history( const long time_step, const double error_signal );
+  //! Create an entry for the given time step at the end of the eprop history.
+  void emplace_new_eprop_history_entry( const long time_step, const bool has_norm_step = true );
```

Comment:
See comment above on name.

Reply:
Fixed in [5356e64](https://github.com/nest/nest-simulator/pull/3207/commits/5356e6478d2f0c2410723d8ce5b146036fa418a7)